src/views/communities/components/AddCommunityDialog.js

```diff
@@ -0,0 +1,343 @@
+import React, { useState, useEffect } from 'react';
+import {
+  Box,
+  Button,
+  TextField,
+  MenuItem,
+  Select,
+  FormControl,
+  InputLabel,
+  Typography,
+  Modal,
+  Divider,
+  IconButton,
+  Avatar,
+  Grid
+} from '@mui/material';
+import PersonIcon from '@mui/icons-material/Person';
+import UploadDark from 'assets/images/icons/upload-dark';
+import UploadLight from 'assets/images/icons/upload-light';
+import { createCommunity } from 'services/createCommunity';
+import { fetchUsers } from 'services/fetchUsers';
+
+const LogoUploader = ({ logoImage, handleLogoUpload }) => (
+  <Box display="flex" alignItems="center" my={3}>
+    {logoImage ? (
+      <Avatar src={URL.createObjectURL(logoImage)} alt="Logo Image" sx={{ width: 80, height: 80, marginRight: 2 }} />
+    ) : (
+      <Avatar sx={{ width: 80, height: 80, marginRight: 2 }}>
+        <PersonIcon />
+      </Avatar>
+    )}
+    <Button variant="outlined" component="label" startIcon={<UploadDark />} sx={{ marginLeft: 2 }}>
+      Upload Logo/Image
+      <input hidden accept="image/png, image/jpeg, image/gif" type="file" onChange={handleLogoUpload} />
+    </Button>
+  </Box>
+);
+
+const CoverUploader = ({ coverImage, handleCoverUpload }) => (
```

Comment:
use ImageUpload component for this, 
as mentioned in above coment

Reply:
Also instead of loading the images, use Material UI icons.